Gemfile

```diff
@@ -81,6 +81,8 @@ group :development, :test do
 
   gem "factory_bot_rails", "~> 6.2.0"
   gem "rspec-rails", "~> 6.0"
+
+  gem "standard-rails"
```

Comment:
I think we need the standard gem and the .standard.yml file.
https://github.com/standardrb/standard#plugins

I could not run bundle exec standardrb without adding the gem and file.

Reply:
It's my mistake not to include `.standard.yml` to this commit. I have it in my local env.

I believe `standard-rails` should depend on `standard`, so I created a PR.
https://github.com/standardrb/standard-rails/pull/21

Reply:
So they say we should add `standard` as a explicit dependency, so we'll do so.